### DISCUSSION TITLE
Opinions on Cloudflare plans

### DISCUSSION THREAD

**Discussion by eafif** [basecamp/kamal#485] [General]
Kamal seems to work best with a CDN such as Cloudflare to serve static assets and handle SSL. As far as I can tell, this can be accomplished using the [Free or the Pro plans](https://www.cloudflare.com/en-ca/plans/). However there is no mention of their Uptime SLA for these plans.

Would be you comfortable using these plans for customer facing applications? Or it is best to use the Business plan even if you aren't using any of the additional features that come with higher plan?

**Comment by misterhtmlcss** [comment]
Like all things it depends on the client. 

I contract with a fintech company, so the money we make per unit of effort is high and cost of errors also very high, so we pay for everything no matter what especially for QoS. 

However I also contract with a medium size business that I work on their internal CRM/marketing platform and other than transactions on a few critical workflows they would prefer free than paid. They mostly (99%-ish) prefer to use free over a .01% chance of failure for anything that isn't $$ in the bank.

I hope this helps.
